src/native/corehost/apphost/resource.windows.h

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef __RESOURCE_H__
+#define __RESOURCE_H__
+
+// We don't want to set an automatically recognized manifest for apphost.
+// Use a non-reserved manifest resource ID. 1-16 are reserved (winuser.h)
+#define IDR_ACTCTX_MANIFEST 101
```

Comment:
I wonder if we should use some higher "Random looking" number. The problem is that apphost gets customized with whatever native resources are present in the managed .dll (which can be pretty much anything), and if there was a conflict "bad things" would happen.
(This also adds additional work into our "make resource update multi-platform" since we will need to be able to truly update, not just create).